sci-misc/mendeleydesktop/mendeleydesktop-1.13.ebuild

```diff
@@ -0,0 +1,114 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit eutils fdo-mime multilib
+
+MY_P_AMD64="${P}-linux-x86_64"
+MY_P_X86="${P}-linux-i486"
+
+DESCRIPTION="A free research management tool for desktop and web"
+HOMEPAGE="http://www.mendeley.com/"
+SRC_URI="amd64? ( ${MY_P_AMD64}.tar.bz2 )
+	x86? ( ${MY_P_X86}.tar.bz2 )
+	amd64-linux? ( ${MY_P_AMD64}.tar.bz2 )
+	x86-linux? ( ${MY_P_X86}.tar.bz2 )"
+
+LICENSE="Mendeley-terms"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RESTRICT="fetch"
+
+DEPEND=""
+RDEPEND="
+	>=dev-qt/qtcore-4.6:4
+	>=dev-qt/qtgui-4.6:4
+	>=dev-qt/qtsvg-4.6:4
+	>=dev-qt/qtwebkit-4.6:4
+	>=dev-qt/qtxmlpatterns-4.6:4"
+
+QA_PREBUILT="/opt/mendeleydesktop/.*"
+
+pkg_nofetch() {
+	elog "Please download ${A} from:"
+	elog "http://www.mendeley.com/download-mendeley-desktop/"
+	elog "and move it to ${DISTDIR}"
+}
+
+src_unpack() {
```

Comment:
*inherit vcs-snapshot* and afterwards calling *src_unpack* **or** calling *vcs-snapshot_src_unpack* explicitly in *src_unpack* results both in *work/mendeleydesktop-1.13-linux-x86_64*. That's not what I want.